assets/root-files/Authentication/phpunit.xml.dist

```diff
@@ -0,0 +1,45 @@
+<phpunit bootstrap="./test/bootstrap.php" colors="true">
+    <testsuites>
+        <testsuite name="zend-authentication Test Suite">
+            <directory>./test/</directory>
+        </testsuite>
+    </testsuites>
+
+    <groups>
+        <exclude>
+            <group>disable</group>
+        </exclude>
+    </groups>
+
+    <filter>
+        <whitelist addUncoveredFilesFromWhitelist="true">
+            <directory suffix=".php">./src</directory>
+        </whitelist>
+    </filter>
+
+    <php>
+        <ini name="date.timezone" value="UTC"/>
+
+        <!-- OB_ENABLED should be enabled for some tests to check if all
+             functionality works as expected. Such tests include those for
+             Zend\Soap and Zend\Session, which require that headers not be sent
+             in order to work. -->
+        <const name="TESTS_ZEND_OB_ENABLED" value="0" />
```

Comment:
It's not just a proxy; it also has the conditional to determine whether or not to enable output buffering. Again, I'm trying to keep as much compatibility with the current testing infrastructure as possible. Considering how long it takes to test a split, at this point, I do not plan on making further changes (as we are planning the split for tomorrow, assuming I can get 2.4.1 released today).